org.greatfire.FreeBrowser.yml

```diff
@@ -0,0 +1,145 @@
+app-id: org.greatfire.FreeBrowser
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.chromium.Chromium.BaseApp
+base-version: '23.08'
+command: freebrowser
+separate-locales: false
+finish-args:
+  - --device=all
+  - --env=GTK_PATH=/app/lib/gtkmodules
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --share=ipc
+  - --share=network
+  - --socket=cups
+  - --socket=pcsc # FIDO2
+  - --socket=pulseaudio
+  - --socket=x11
+  - --socket=wayland
```

Comment:
it is either x11 alone or wayland + fallback-x11. not both without a good reason.

Reply:
two wrongs don't make a right.

Reply:
two wrong don't make a right.